superset/assets/src/chart/Chart.jsx

```diff
@@ -35,7 +35,12 @@ const propTypes = {
   actions: PropTypes.object,
   chartId: PropTypes.number.isRequired,
   datasource: PropTypes.object.isRequired,
-  filters: PropTypes.object,
+  // original selected values for FilterBox viz
+  // so that FilterBox can pre-populate selected values
+  // only affect UI control
+  origSelectedValues: PropTypes.object,
```

Comment:
How about `originalFilters`?
On a side note, is this field only used by `filter_box`?

Reply:
I verified and only `table` and `filter_box` use this `filters` field from `ChartProps`.

In a follow up thought, `initialValues` might be a better name (use `values` from your initial idea to make it sounds more open to other vis ` and `initial` to shorten the `origSelected` part)

Reply:
vote for `initialValues`